Update image and language (#6498)
dbt-labs/docs.getdbt.com: website/docs/docs/cloud/git/connect-gitlab.md

## For dbt Cloud Developer and Team tiers

To connect your GitLab account:
1. From dbt Cloud, click on your account name in the left side menu and select **Account settings**. 
2. Select **Personal profile** under the **Your profile** section.
3. croll down to **Linked accounts**.

> 3. Click **Link** to the right of your GitLab account.

Author: ```suggestion
4. Click **Link** to the right of your GitLab account.
```

> 3. croll down to **Linked accounts**.

Reviewer: ```suggestion
3. Scroll down to **Linked accounts**.
```

> <Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-github/github-connect.png" title="Navigated to Linked Accounts under your profile"/>

Reviewer: This might read a little better for accessibility purposes

```suggestion
<Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-github/github-connect.png" title="The Personal profile settings with the Linked Accounts section of the user profile"/>
```
